exports.getListUser = async (req, res, next) => {
    const error = new Error();

    try {
        
    } catch (err) {
        next(err);
    }
};